src/test/suite/fixtures/recorded/languages/scss/changeName3.yml

```diff
@@ -0,0 +1,29 @@
+languageId: scss
+command:
+  version: 1
+  spokenForm: change name
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: name, includeSiblings: false}
+initialState:
+  documentContents: |
+    a {
+      margin: 10px 10px 10px 10px;
+    }
+  selections:
+    - anchor: {line: 1, character: 14}
+      active: {line: 1, character: 14}
+  marks: {}
+finalState:
+  documentContents: |
+    a {
+      : 10px 10px 10px 10px;
```

Comment:
I almost feel like this should be "key", and "name" should be the selector, but I'm really not sure. Usually "name" is the name in a declaration